miniscope_io/stream_daq.py

```diff
@@ -235,11 +241,16 @@ def _uart_recv(
         log_uart_buffer = BitArray([x for x in uart_bites])
 
         try:
-            while 1:
+            while not self.terminate.is_set():
```

Comment:
I think this is my leftover from testing direct termination of methods and doesn't have an effect now, so I reverted.